README.md

```diff
@@ -106,6 +106,9 @@ DATABASE_URL=
 NODE_ENV=
 PGSSLMODE=no-verify
 EMAIL_ADMIN=
+NODEMAILER_SECRET=
+NODEMAILER_EMAIL=aceleradorametodosageis@gmail.com
+NODEMAILER_PASSWORD=VDR1ehc_eha1cgj7bxw
```

Comment:
Esse valor é falso ou é o mesmo que será usado em produção? Se for o valor de produção, não deveria ser commitado por questões de segurança.